apps/prairielearn/src/lib/db-types.ts

```diff
@@ -548,6 +548,7 @@ export const GroupConfigSchema = z.object({
   minimum: z.number().nullable(),
   name: z.string().nullable(),
   student_authz_create: z.boolean().nullable(),
+  student_authz_group_name: z.boolean(),
```

Comment:
Nit: if we're thinking of "name" as the verb, should this column be named `student_authz_name`? That would seem to match the JSON property (`studentGroupName`).

Reply:
Renamed to student_authz_choose_name in d561cda6004fa9af2ac9178df6ae36e68b9cab92.